lua/starfall/libs_sv/wire.lua

```diff
@@ -308,6 +308,43 @@ function wire_library.adjustOutputs(names, types)
 	WireLib.AdjustSpecialOutputs(ent, names, types)
 end
 
+--- Creates/Modifies wire inputs/outputs. All wire ports must begin with an uppercase
+-- letter and contain only alphabetical characters.
+-- @param inputs A key-value table with input port names as keys and types as values. Can be nil to not affect input ports.
+-- @param outputs A key-value table with output port names as keys and types as values. Can be nil to not affect output ports.
+function wire_library.adjustPorts(inputs, outputs)
+	if inputs then checkluatype(inputs, TYPE_TABLE) end
+	if outputs then checkluatype(outputs, TYPE_TABLE) end
+
+	if inputs then
+		local names = {}
+		local types = {}
+
+		for n,t in pairs( inputs ) do
+			if type(n) == "number" then SF.Throw("Invalid parameter, expected a key-value table") end
```

Comment:
`if type(n) ~= "string" or type(t) ~= "string" then SF.Throw("Expected string string key value pairs, got a " .. type(n) .. " " .. type(t) .. " pair.", 2) end`

Reply:
Was not sure with that, will change to that